docs/aerodromes/classd/Bankstown.md

```diff
@@ -82,14 +88,14 @@ Due to the close proximity in critical stages of flight, there are additional co
 At Bankstown, all Runways are considered Independent unless a Multi-Engined OR Jet aircraft departs from an adjacent Runway. 
 In this instance, runways are treated as dependent *for departures ONLY*. Controllers should apply the same Runway standard across all dependant Runways. Arriving aircraft are exempt from this clause.
 
-Definitions;
-Independent: Runways can operate simultaneously without restriction.
-Dependent: Runways are operated with restrictions, often one at a time.
+**Independent**: Runways can operate simultaneously without restriction.  
+**Dependent**: Runways are operated with restrictions, often one at a time.
```

Comment:
often one at a time? Can we define that?

Reply:
Attempted definitions using the examples below... You're welcome word something up, everything I tried was really wordy

Reply:
I'm assuming that in these cases, its a temporary arrangement for the duration of an aircraft's arrival/departure and that the ATIS doesn't change? If so, I'd imagine that common sense would dictate which runway is available and the traffic levels we see online wouldn't cause too many issues in using that single runway for the short period of time.